scripts/build

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# DESC: Build the Harvester by running scripts/default
```

Comment:
I think it builds the binaries for harvester, harvester-webhook and upgrade-helper. Comment should reflect that accurately, IMO.